src/uu/tail/src/args.rs

```diff
@@ -282,19 +282,13 @@ impl Settings {
             }
         }
 
-        let mut inputs: VecDeque<Input> = matches
-            .get_many::<String>(options::ARG_FILES)
-            .map(|v| v.map(|string| Input::from(&string)).collect())
-            .unwrap_or_default();
+        settings.inputs = matches
+            .get_raw(options::ARG_FILES)
```

Comment:
What's the disadvantage of `get_raw`?

Reply:
Not sure there's a real disadvantage, but it feels like a workaround. Something that should only be necessary if you have parsed values and want to access the raw value as well.

Reply:
I'm pretty sure it's meant as the direct way to access the parsed values as `OsStr` ;) imho value parsing to get an `OsStr` from clap is just a waste of resources.

Reply:
I'd be surprised if it actually did any parsing. It's just gonna expect an OsStr, which always succeeds and then can give us an OsStr as a value.

Reply:
Can we do it the easy and direct way, please? From the docs of clap `get_raw`

> Iterate over the original argument values.

With `get_raw`, there's nothing else between us and the `OsStr`, neither during compile time nor runtime and the code is clearly expressing that intent.

Reply:
That intent is _exactly_ what an `OsStr` value parser expresses and it does it i n the clap builder, instead of at this location. It is in my opinion the most easy and direct way, because it uses claps normal parsing.

Besides: https://github.com/clap-rs/clap/issues/3809

Reply:
That clap issue is open waiting for decision since last year, so I think we can safely use it. Can we settle this please? If you insist I'll change it somehow, but I really don't think it's worth it.

Reply:
Fine. Let's hope I can make a better API than clap's if I ever find time to work on my own parser again :smile